src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
+    price -= 20;
+  }
```

Comment:
Prefer 'if' with 'return' over 'if else' to simplify the conditions. Also, avoid adding 'else' after 'if' with 'return'.